contracts/scripts/core/DeployCore.s.sol

```diff
@@ -15,6 +15,12 @@ import {BlockTracker} from "../../contracts/core/BlockTracker.sol";
 import {console} from "forge-std/console.sol";
 
 contract DeployTestnet is Script {
+
+    // Amount of ETH required to fund the Oracle contract.
+    uint256 public constant ORACLE_FUNDING = 1000 ether;
```

Comment:
1000 ether is not a feasible initial funding amount for the oracle account in prod. This would mean our team would have to lock 1000 eth in the L1 bridge contract upon chain genesis. See comparable `RELAYER_INITIAL_FUNDING` which is 1 eth.

Reply:
I assume that this is for testnet/devnet only, since the following is also not a feasible initial amount: https://github.com/primev/mev-commit/blob/main/contracts/scripts/standard-bridge/DeployStandardBridge.s.sol#L17

Reply:
`ORACLE_FUNDING` should be 1-5 eth for all of  testnet/devnet/mainnet from what I see, we'll probably need to withdraw funds to the oracle address periodically

Reply:
Fixed.